swaybar/tray/item.c

```diff
@@ -115,10 +115,11 @@ static int get_property_callback(sd_bus_message *msg, void *data,
 	const char *prop = d->prop;
 	const char *type = d->type;
 	void *dest = d->dest;
+	sway_log_importance_t log_lv = strcmp(prop, "IconThemePath") ? SWAY_ERROR : SWAY_DEBUG;
```

Comment:
Can we also check the value of `sd_bus_message_get_errno()` as well here? Just to make sure the error is because of a missing property and not something else.

Reply:
I updated the code. Now using the `name` member of the error object to determine whether this error is about missing property or not. I didn't use `sd_bus_message_get_errno()` because I'm not sure if the error number will be changed or not in the future.